app/models/content_item.rb

```diff
@@ -82,6 +82,25 @@ def as_json(options = nil)
     end
   end
 
+  # We store the description in a hash because Publishing API can send through
+  # multiple content types.
+  def description=(value)
+    super("value" => value)
+  end
+
+  def description
+    description = super
+
+    if description.is_a?(Hash)
+      description.fetch("value")
+    else
+      # This is here to ensure backwards compatibility during data migration:
+      # db/migrate/20151130111755_description_value_hash.rb
+      # It can be removed afterwards.
```

Comment:
Have to taken into consideration the fact that the database change happens before the services are restarted?  I don't know how this affects migrations with mongo.